_resources/shared/violence/strong-hearts.md

```diff
@@ -0,0 +1,15 @@
+---
+title: StrongHearts Native Helpline
+href: https://strongheartshelpline.org/
+category: violence
+phones:
+  primary:
+    display: 1-844-7NATIVE (762-8483)
+    link: 18447628483
+---
+
+StrongHearts Native Helpline is a 24/7 safe, confidential and anonymous domestic, dating and sexual violence helpline for American Indians and Alaska Natives, offering culturally-appropriate support and advocacy.
+
+You can also [visit their website](https://strongheartshelpline.org/) and tap the chat button to chat with someone if calling doesn't work for you.
+
+StrongHearts advocates are mandatory reporters, but as an anonymous helpline, they do not have the information necessary to make reports. **If you want to avoid a report being made, please stay anonymous.**
```

Comment:
I confirmed this by emailing their communications director.

Reply:
You're a superstar